app/models/review.rb

```diff
@@ -0,0 +1,8 @@
+class Review < ApplicationRecord
+  validates :title, presence: true
+  validates :rating, numericality: { minimum: 0, maximum: 5 }
+  validates :heatIndex, numericality:{minimum: 0, maximum: 10}
+  belongs_to :sauce
+  
+
```

Comment:
delete either line 6 AND line 7, or at least both 
add a space after `numericality:` on line 4